site/_config.yml

```diff
@@ -126,9 +126,9 @@ host:    127.0.0.1
 sourcerepourl: https://github.com/apache/incubator-edgent
 downloadsurl: /docs/downloads
 projurl: /
-#docsurl: /javadoc/latest
+docsurl: /javadoc/latest
 # above yields things like the following when running on a "jekyll serve" test server
 # [2016-12-22 15:23:31] ERROR `/favicon.ico' not found.
 # [2016-12-22 15:33:17] ERROR `/javadoc/latest/resources/fonts/dejavu.css' not found.
 # the following is undesirable as it doesn't link to the "jekyll serve" test server
-docsurl: https://edgent.incubator.apache.org/javadoc/latest
+# docsurl: https://edgent.incubator.apache.org/javadoc/latest
```

Comment:
Yeah, I thought the javadoc looked correct too but wasn't confident they were completely benign.  But even if benign, I think it's a bad idea to commit something that's generating them.  At best folks will be later be surprised / confused by them.